.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,14 @@
-This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.
+Thanks for your interest in The Odin Project. In order to get PRs closed in a reasonable amount of time, we request that you include a baseline of information about hthe changes you are proposing. Please answer the following triage quesions:
+
+### Describe the nature of the changes made:
+
+...your text here
+
+### If this PR is related to an open issue please reference it with the `#` operator and the issue number below:
+
+...issue
+
+### If not, please open a new issue describing the problems this PR solves
 
 Thank you,
 
```

Comment:
Missing "The Odin Project team", like in the Issue template

Reply:
ON further review, it seems it's there in the original file, but not showing here.

Reply:
It's not missing, it just wasn't a change